### DISCUSSION TITLE
Prognostic aerosol cloud evaporation

### DISCUSSION THREAD

**Discussion by jchylik** [ARM-Development/comble-mip#66] [Q&A] [answered]
So far it is unclear what happens with prognostic aerosols during cloud evaporation.
In Part II:  Small-domain LES, prognostic aerosol and ice, we are trying to simulate conditions using the three aerosols modes. The cloud droplet nucleation then logically acts as a sink. However, does it mean that every aerosol particle is fully lost? Considering that clouds are in Wegener–Bergeron–Findeisen regime, that might be quite unfortunate. Or do the aerosols "return" when cloud droplets evaporate?

Base on the description in [aerosol-specification.html](https://arm-development.github.io/comble-mip/notebooks/setup/aerosol-specification.html), there seems to be two main options:

a) Recovery: When cloud droplet number decreases due to evaporation, a proportional number is **added** to one of the aerosol modes (Mode 1 ? Or mode 2?)

b) No recovery:  When cloud  droplet number decreases due to evaporation, the number of aerosols in all three nodes stays constant.

**Comment by fridlind** [comment, marked as answer]
The [aerosol specification page](https://arm-development.github.io/comble-mip/notebooks/setup/aerosol-specification.html) is intended to describe the initial size distributions (vertically uniform number mixing ratios in each mode).

The [main model configuration page](https://arm-development.github.io/comble-mip/main_configuration.html#) is intended to describe how to run the model for the prognostic aerosol setting, where a footnote to the prognostic aerosol section heading indicates our expectation that "prognostic aerosol schemes will account for number losses to collision-coalescence (e.g., rain formation and riming) and potentially others (e.g., dry deposition)". This is consistent with the recovery option, which allows for tracking of collisional losses of aerosol numbers. How a model handles that mode-wise would depend on how the aerosol scheme is implemented; we are not specifying that.

Thank you for the question! Please let us know if anything remains unclear or if you recommend changes to the specification pages to better clarify anything.

**Reply by jchylik** [reply]
> This is consistent with the recovery option, which allows for tracking of collisional losses of aerosol numbers.

Oh, OK, thank you for the clarification. 

> How a model handles that mode-wise would depend on how the aerosol scheme is implemented; we are not specifying that.

I see, that leave quite a lot of leeway, but it expect that it would not change the results too much
